services/ingestion/etl/2014tc16i5cb005/csv/src/lib/transform.js

```diff
@@ -0,0 +1,280 @@
+// @flow
+
+import countries from 'i18n-iso-countries';
+import getCountryCode from '@eubfr/lib/getCountryCode';
+import numeral from 'numeral';
+import sanitizeBudgetItem from '@eubfr/lib/budget/budgetFormatter';
+import type { Project } from '@eubfr/types';
+
+/*
+ * Transform message (2014tc16i5cb005 CSV)
+ */
+
+/**
+ * Preprocess `budget`.
+ *
+ * Input fields taken from the `record` are:
+ *
+ * - `Project final contract amount`
+ * - `Project EU co-financing rate`
+ *
+ * @memberof 2014tc16i5cb005CsvTransform
+ * @param {Object} record The row received from parsed file
+ * @returns {Budget}
+ */
+
+const getBudget = record => {
+  const { _value: total } = numeral(record['Project final contract amount']);
+
+  const { _value: percentage } = numeral(
+    record['Project EU co-financing rate']
+  );
+
+  const euContrib = total * percentage;
```

Comment:
Sometimes the data are wrongly formatted...

In the CSV file, I find 2 occurrences where `Project EU co-financing rate` is not a rate, but the absolute value directly: `500665.00%` (85% of `589017.00`) and `372347.00%` (85% of `438055.00`). 

Could we add a bit of logic here, something like "if `percentage` is greater than 1 (or 100%), then it's `euContrib = percentage * 100` directly"?